be/src/storage/lake/tablet_manager.cpp

```diff
@@ -345,7 +345,9 @@ Status TabletManager::create_tablet(const TCreateTabletReq& req) {
     RETURN_IF_ERROR(starrocks::convert_t_schema_to_pb_schema(
             req.tablet_schema, next_unique_id, col_idx_to_unique_id, tablet_metadata_pb->mutable_schema(),
             req.__isset.compression_type ? req.compression_type : TCompressionType::LZ4_FRAME));
-    RETURN_IF_ERROR(create_schema_file(req.tablet_id, tablet_metadata_pb->schema()));
+    if (req.create_schema_file) {
```

Comment:
```suggestion
    if (req.__isset.create_schema_file && req.create_schema_file) {
```

Reply:
The thrift field `create_schema_file` is defined with default value `true`